src/core/store/modules/collateral/collateral.reducer.ts

```diff
@@ -144,7 +144,7 @@ const collateralReducer = createReducer(collateralInitialState, (builder) => {
         id: escrowId,
         cratio: '0',
         collateralValue: '0',
-        minCRatio: Math.round(Number(cratio.toString())) / 100,
+        minCRatio: Math.round(100 * Number(cratio.toString())) / 100,
```

Comment:
why not just `Number(cratio.toString()))`?

Reply:
We need to round `cratio` to 2 decimal places so the formatting isn't insane. We don't want to be showing `Collateralization Ratio: 97.34579101%`

`Math.round()` rounds to the nearest integer, so we have to multiply the original number by 100 first so that we get 2 decimal places in the final result.

Reply:
gotcha, can u add an inline comment saying that plz